client/lib/parsing.rs

```diff
@@ -549,15 +573,13 @@ pub(super) fn parse_payment_info(
     })
 }
 
-pub(crate) fn get_transfer_target(target_account: &str) -> Result<TransferTarget> {
-    let account = PublicKey::from_hex(target_account).map_err(|error| {
-        Error::InvalidArgument(
-            "target_account",
-            format!("failed to parse as a public key: {}", error),
-        )
+pub(crate) fn get_target_account(target_account: &str) -> Result<PublicKey> {
```

Comment:
We can maybe even rename this function to `parse_public_key`. If I'm not mistaken it is exactly what it does :)

Reply:
+1 from me on the rename